tox.ini

```
@@ -0,0 +1,20 @@
+[tox]
+envlist = py27,
+          py38,
+minversion = 1.6
+
+[testenv]
+deps = py27: pytest < 5.0
+       py38: pytest
+       pytest-cache
+       pytest-cov
+       pytest-flake8
+       .[test]
+commands = py.test \
```

```suggestion
commands = pytest \
```

Sadly this is the spelling the pytest developers have agreed on. (https://github.com/tox-dev/tox/issues/484)

---

:+1: